networkx/drawing/nx_pylab.py

```diff
@@ -402,6 +403,9 @@ def draw_networkx_nodes(
     label : [None | string]
         Label for legend
 
+    margins : [None | list] (default=None)
+        Horizontal and vertical plot margins. List should contain exactly two elements.
```

Comment:
```suggestion
    margins : Sequence of 2 numbers or None (default=None)
        The sequence contains horizontal and vertical axis margins. Adjust to avoid image being clipped.
```